Pages/ChildSaveBank.js

```diff
@@ -0,0 +1,58 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+
+function ChildSaveBank({ navigation }) {
+    return (
+    <View style={styles.container}>
+         <Button 
+            style={styles.BackButton}
+            title="<--"
+            onPress={() => navigation.goBack()}
+        />
+        <Text>Save Bank:</Text>
+      
+        <Button 
+            style={styles.NewSaveTransaction}
+            title="New Transaction"
+            onPress={() => navigation.navigate('EnterSaveTransaction')}
```

Comment:
Best coding practice, put this in an object. https://stackoverflow.com/questions/4286899/javascript-enum-to-corresponding-string-value. Makes it easier to debug